lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -886,29 +887,34 @@ def import_workflow( self, trans, cntrller='workflow', **kwd ):
                             pass
                     if tool_shed_url:
                         # We've received the textual representation of a workflow from a Galaxy tool shed.
-                        message = "Workflow <b>%s</b> imported successfully." % escape( workflow.name )
+                        message = "Workflow %s imported successfully." % escape( workflow.name )
                         url = '%s/workflow/view_workflow?repository_metadata_id=%s&workflow_name=%s&message=%s' % \
                             ( tool_shed_url, repository_metadata_id, encoding_util.tool_shed_encode( workflow_name ), message )
                         return trans.response.send_redirect( url )
                     elif installed_repository_file:
                         # The workflow was read from a file included with an installed tool shed repository.
-                        message = "Workflow <b>%s</b> imported successfully." % escape( workflow.name )
+                        message = "Workflow %s imported successfully." % escape( workflow.name )
                         if cntrller == 'api':
                             return status, message
                         return trans.response.send_redirect( web.url_for( controller='admin_toolshed',
                                                                           action='browse_repository',
                                                                           id=repository_id,
                                                                           message=message,
                                                                           status=status ) )
-                    return self.list( trans )
+                    redirect_url = url_for( '/' ) + 'workflow?status=' + status + '&message=%s' % escape( message )
+                    return trans.response.send_redirect( redirect_url )
         if cntrller == 'api':
             return status, message
-        return trans.fill_template( "workflow/import.mako",
-                                    url=url,
-                                    message=message,
-                                    status=status,
-                                    use_panels=True,
-                                    myexperiment_target_url=myexperiment_target_url )
+        if status == 'error':
+            redirect_url = url_for( '/' ) + 'workflow?status=' + status + '&message=%s' % escape( message )
+            return trans.response.send_redirect( redirect_url )
+        else:
+            return {
+                'url' : url,
+                'message' : message,
+                'status' : status,
+                'myexperiment_target_url' : myexperiment_target_url
```

Comment:
FYI, spaces before a colon `:` are discouraged. We don't enforce [E203](http://pycodestyle.pycqa.org/en/latest/intro.html#error-codes) in our style guidelines, but only because we allow to align the dictionary values.

Not a merge blocker anyway.